install error

OS: Mac v12.5
Python: 3.10.6

Install script is throwing this error in the seed dcs stage.
```
$ python --version
Python 3.10.6
$ make install
...
python -m ambuda.seed.dcs
Fetching latest data ...
Cloning into '/Users/kvic/src/ambuda/data/ambuda-dcs'...
remote: Enumerating objects: 47, done.
remote: Counting objects: 100% (34/34), done.
remote: Compressing objects: 100% (25/25), done.
remote: Total 47 (delta 21), reused 20 (delta 9), pack-reused 13
Receiving objects: 100% (47/47), 10.10 MiB | 96.00 KiB/s, done.
Resolving deltas: 100% (25/25), done.
Already on 'main'
Your branch is up to date with 'origin/main'.
HEAD is now at 7622c22 Fix double tab in meghadutam parse data
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.10/3.10.6_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/homebrew/Cellar/python@3.10/3.10.6_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/Users/kvic/src/ambuda/ambuda/seed/dcs.py", line 123, in <module>
    run()
  File "/Users/kvic/src/ambuda/ambuda/seed/dcs.py", line 102, in run
    add_parse_data(path.stem, path)
  File "/Users/kvic/src/ambuda/ambuda/seed/dcs.py", line 87, in add_parse_data
    for slug, blob in iter_parse_data(path):
  File "/Users/kvic/src/ambuda/ambuda/seed/dcs.py", line 68, in iter_parse_data
    raise ValueError(f'Line "{line}" must have exactly three fields.')
ValueError: Line "gOtamaH	gOtama	pos=n,g=m,c=1,n=s" must have exactly three fields.
make[1]: *** [db-seed-basic] Error 1
make: *** [install] Error 2```

--- a/ambuda/seed/dcs.py
+++ b/ambuda/seed/dcs.py
@@ -64,8 +64,8 @@ def iter_parse_data(path: Path):
                     xml_id = value
                     _, _, block_slug = xml_id.partition(".")
             elif line:
-                if line.count("\t") != 3:
-                    raise ValueError(f'Line "{line}" must have exactly three fields.')
+                if line.count("\t") != 2:
+                    raise ValueError(f'Line "{line}" must have exactly two tabs.')
                 buf.append(line)
             else:
                 yield block_slug, "\n".join(buf)

--- a/ambuda/utils/cheda.py
+++ b/ambuda/utils/cheda.py
@@ -60,6 +60,7 @@ LAKARAS = {
     "lrt": "simple future",
     "lot": "imperative",
     "lan": "imperfect",
+    "lan_unaug": "imperfect (unaugmented)",
     "vidhilin": "optative",
     "ashirlin": "benedictive",
     "lun": "aorist",

--- a/test/ambuda/utils/test_parsing.py
+++ b/test/ambuda/utils/test_parsing.py
@@ -11,6 +11,10 @@ from ambuda.utils.cheda import readable_parse
         ("pos=va,g=n,c=3,n=p", "participle, neuter instrumental plural"),
         ("pos=v,p=3,n=s,l=lat", "verb, third-person singular present"),
         ("pos=v,p=2,n=s,l=ashirlin", "verb, second-person singular benedictive"),
+        (
+            "pos=v,p=3,n=s,l=lan_unaug",
+            "verb, third-person singular imperfect (unaugmented)",
+        ),
         ("pos=n,g=m,comp=y", "noun, compounded"),
         ("pos=i", "indeclinable"),
     ],

